README.rst

```diff
@@ -5,20 +5,35 @@ ORM plugin for using Cloud Spanner as a `database backend
 <https://docs.djangoproject.com/en/2.2/ref/databases/#using-a-3rd-party-database-backend>`__
 for Django.
 
+How it works
+------------
+
+Overall design
+~~~~~~~~~~~~~~
+
+.. figure:: ./assets/overview.png
+   :alt:
+
+Internals
+~~~~~~~~~
+
+.. figure:: ./assets/internals.png
+   :alt:
+
 
 Installation
 ------------
 
-To use this library you'll need a Google Cloud Platform project with the Cloud
-Spanner API enabled. See the `Cloud Spanner Python client docs
+Using this library requires a Google Cloud Platform project with the Cloud
+Spanner API enabled. See the Spanner' Python Client `documentation
```

Comment:
```suggestion
Spanner API enabled. See the Spanner Python client `documentation
```

I'm no usage expert, but I think "Spanner" and "Python" are capitalized here as separate terms -- not as part of "Spanner Python Client" -- and "client" can be lowercase.

Reply:
👍 Done.